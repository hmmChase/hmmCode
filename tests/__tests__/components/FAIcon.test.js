import { render, prettyDOM, fireEvent } from '@testing-library/react';
import FAIcon from '../../components/FAIcon';

describe('FAIcon', () => {
  it('matches snapshot', () => {
    const mockProps = {};

    const utils = render(<FAIcon {...mockProps} />);

    const container = utils.container.firstChild;

    expect(container).toMatchSnapshot();
  });

  it('has correct class', () => {
    const mockProps = {};

    const utils = render(<FAIcon {...mockProps} />);

    const icon = utils.container.firstChild;

    expect(icon).toHaveClass('fa-thumbs-up');
  });
});
