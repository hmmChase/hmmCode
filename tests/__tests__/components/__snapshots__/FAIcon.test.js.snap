// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAIcon matches snapshot 1`] = `
<i
  class="fa-thumbs-up"
/>
`;
